JASP-Engine/JASP/R/multinomialtest.R

```diff
@@ -643,7 +643,13 @@ MultinomialTest <- function (dataset = NULL, options, perform = "run",
     .quitAnalysis(paste0(variable, "variable contains negative values"))
   }
 
-  if (!all(counts == round(counts))) {
+  # only applies for observed counts, expected counts can be proportions
+  if (!expectedCounts & !all(counts == round(counts))) {
     .quitAnalysis(paste0(variable, "variable must contain only integer values."))
   }
+
+  # a vector which is too big, ittakes up too much memory
+  if (sum(counts > 6e8)) {
```

Comment:
What do you base `6e8` on?

(Is it completely necessary to allocate a vector of size `sum(counts)`?)